### PR TITLE
Added some content and basic styling

### PR DESCRIPTION
**Pull request by ThomasKirst**
Please check the styling!
Thanks :)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,17 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <h1>Quiz App 123 456</h1>
+  <h1>Quiz App</h1>
```
Comment (miriamhaenle) [inline src/index.html:10]: Consider wrapping your `h1 `in a `header` element

File: src/styles.css [hunk 2]
```diff
@@ -1,3 +1,9 @@
 body {
   font-family: sans-serif;
 }
+
+article {
+  border: 2px solid black;
```
Comment (miriamhaenle) [inline src/styles.css]: 🙀 This is not good user experience. Please make the card hotpink!

**Review by miriamhaenle — CHANGES_REQUESTED** [review]
Please review my comments

File: src/index.html (hunk 1 above)
Comment (ThomasKirst) [reply src/index.html:10]: Won't fix

File: src/styles.css (hunk 2 above)
Comment (ThomasKirst) [reply src/styles.css]: Good catch! Thanks, fixed it! :)

**Review by miriamhaenle — APPROVED** [review]
Great job!

**Pull request merged**